### DISCUSSION TITLE
Allow branch to be selected on Project creation

### DISCUSSION THREAD

**Discussion by epomatti** [vercel/community#393] [Idea]
Hi

When creating the project for the first time I can't find a way to change the **branch**.

![image](https://user-images.githubusercontent.com/6530769/156081698-a826ad66-7902-45f4-a045-88f973e22e6e.png)

**Comment by leerob** [comment]
Hey! When cloning a template, there isn't a way inside Vercel to change the branch. You would need to either change the branch name in the source repo on git before or after deployment.

**Reply by epomatti** [reply]
What about this https://docs.gitlab.com/ee/api/branches.html#list-repository-branches
